lib/gui/components/safe-webview/safe-webview.js

```diff
@@ -173,7 +173,10 @@ class SafeWebview extends react.PureComponent {
   static newWindow(event) {
     const url = new URL(event.url);
 
-    if (url.protocol === 'http:' || url.protocol === 'https:') {
+    if (_.every([
+      url.protocol === 'http:' || url.protocol === 'https:',
+      event.disposition === 'foreground-tab'
```

Comment:
Is there somewhere that lists these different dispositions, and documents what actions trigger each disposition?